### PR TITLE
fix Int value range is incorrect 

### PR DESCRIPTION
**Pull request by moneytoyang**
fix int value range is incorrect #826

### CODE REVIEW

File: src/parser/parser.yy [hunk 1]
```diff
@@ -451,8 +451,18 @@ go_sentence
 
 step_clause
     : %empty { $$ = new StepClause(); }
-    | INTEGER KW_STEPS { $$ = new StepClause($1); }
-    | KW_UPTO INTEGER KW_STEPS { $$ = new StepClause($2, true); }
+    | INTEGER KW_STEPS {
+        if ((uint64_t)$1 == 9223372036854775808ULL) {
+            LOG(ERROR) << "Integer overflow";
```
Comment (laura-ding) [inline src/parser/parser.yy]: We need to handle exceptions

File: src/parser/scanner.lex [hunk 2]
```diff
@@ -285,8 +285,11 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
                                         yyterminate();
                                     }
                                 }
-                                int64_t val = 0;
+                                uint64_t val = 0;
```
Comment (laura-ding) [inline src/parser/scanner.lex]: Hex and octal do not need to be processed

File: src/parser/parser.yy (hunk 1 above)
Comment (steppenwolfyuetong) [reply src/parser/parser.yy]: Where is the exception?

Comment (moneytoyang) [reply src/parser/parser.yy]: Where is the exception?

File: src/parser/scanner.lex (hunk 2 above)
Comment (moneytoyang) [reply src/parser/scanner.lex]: why？ Don't they overflow?

File: src/parser/parser.yy (hunk 1 above)
Comment (moneytoyang) [reply src/parser/parser.yy]: Where is the exception?

**Comment by laura-ding** [comment]
I have fixed this problem on p #1181, so close this

**Pull request closed without merging**